tests/emukit/examples/simple_gp_model.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from emukit.examples.simple_gp_model.simple_gp_model import SimpleGaussianProcessModel
+
+
+def test_simple_gp_model_optimize():
+    x = np.linspace(-1, 1, 3)[:, None]
+    y = x **2
+    gp = SimpleGaussianProcessModel(x, y)
+    gp.optimize()
```

Comment:
this test case is missing any assertions

Reply:
I've redone the tests